### DISCUSSION TITLE
Create Share-Link for uploaded files

### DISCUSSION THREAD

**Discussion by MaxTratnikA4VR** [Elringus/UnityGoogleDrive#94] [Q&A]
I managed to create and upload files using the tool (which was really easy and nicely described) but could not find a way to create a shared link from the uploaded files for others to access it (like you can do in Google Drive in the browser). Is there any way to expose a file via the shared-for-everyone-with-the-link method from Unity?

**Comment by elringus** [comment]
Try looking for the related method in the drive public API: https://developers.google.com/drive/api/v3/reference/ If there is one, check in which category it is and whether it's implemented by this plugin.
